### PR TITLE
[stable/horovod] Add chart for horovod

### PR DESCRIPTION
**Pull request by cheyang**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/horovod/templates/service.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  ports:
+  - name: ssh 
+    port: {{ .Values.ssh.port }}
+    targetPort: {{ .Values.ssh.port }}
+  selector:
+    app: {{ template "horovod.name" . }}
+    release: {{ .Release.Name }}
+    role: worker
+
+---
```
Comment (unguiculus) [inline incubator/horovod/templates/service.yaml]: Separate files per resource, please.

File: incubator/horovod/values.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Default values for horovod.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+useHostNetwork: false
+useHostPID: false
+
+ssh:
+  port: 22
+  hostKey: |-
```
Comment (unguiculus) [inline incubator/horovod/values.yaml]: Don't add keys to the chart.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please make list indentation consistent throughout the chart. Either use zero or two spaces.

Please move the chart to stable.

File: incubator/horovod/templates/service.yaml (hunk 1 above)
Comment (cheyang) [reply incubator/horovod/templates/service.yaml]: Sure.

File: incubator/horovod/values.yaml (hunk 2 above)
Comment (cheyang) [reply incubator/horovod/values.yaml]: Thx for reminding.

**Comment by unguiculus** [comment]
Why don't you use the official image? Also, the latest release is v0.13.0 while your versioning is different?

**Comment by cheyang** [comment]
Sure, let me change to the official image.

I was asking them to provide official docker image, because they only provided Dockerfile before.  :)  see https://github.com/uber/horovod/issues/261 .

**Review by unguiculus — APPROVED** [review]

**Pull request merged**